_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,168 @@
+	 1. The Recording Secretary will be nominated by the Managing editors from among the associate editors.
+   1. Appointments Terms
+      1. Appointment for all positions are for three years, which may be extended for an unlimited 
+	 number of terms by a vote of two/thirds of the remaining Editorial Board. If the terms of more 
+         than one third of the Editorial Board end at the same time, all may remain in their positions for 
+         long enough for additional editors be appointed, or one month, which ever comes first. 
+      2. At the time the bylaws are adopted, the three managing editorial terms will be for two, three, and four years,
+         determined by mutual agreement or random chance if no agreement is reached.
+   1. Voting
+      1. All votes described in the bylaws are tallied based on the fraction
+         of the Editorial Board responding within one week written notice
+         via e-mail or other voting mechanism approved by the managing
+         editors. In exceptional circumstances advance notice may be given
+         of a rush decision, where the board may be given one week (or more)
+         notice that their votes will be needed within a shorter window,
+         such as a 12 hour window.
+
+1. Conflicts of interest
+   1. A 'conflict of interest' is defined as any of:
+      1. being a close co-author on a paper together for up to 48 months after publication. "Close co-author" is defined as serving as corresponding author and/or first author together, or equivalent close collaboration.  This definition was chosen to avoid overly restrictive limitations for papers with large numbers of loosely-associated contributing authors, and will have some leeway for interpretation as outlined below.
+      1. a co-awardee on a grant for up to 48 months after the grant, or that currently have a cooperative pending grant.
+      1. if there exists a postdoctoral or graduate advisor relationship at any time in the past.
+      1. at the same institution, or if there is a potential change of institution in progress that would result in being at the same institution.
+      1. having a business relationship (such as serving as a consultant for or part owner of same company) within 48 months.
+
+   1. Editors will try to avoid conflicts of interest with the authors when assigning manuscripts to reviewers.  If the reviewer notices a conflict of interest, they will report it to the editor so the manuscript may be reassigned. If they have any co-authorships with any author within the last 48 months that they consider not close co-authorship, they will report these to the assigning editor for the associate editor for approval as not being "close co-authorships".
+
+   1. If it is later discovered that a reviewer has a conflict of interest with an author during the review process, this review will be discarded, and another review will be sought.
```

Comment:
I still don't like the "conflict of interest with an author" term, as it's not a conflict WITH the author, it's a conflict of interests WITHIN the reviewer himself/herself.